### PR TITLE
Add check to Makefile for clouds.yaml presence

### PR DESCRIPTION
**Pull request by fdobrovolny**

### CODE REVIEW

File: terraform/Makefile [hunk 1]
```diff
@@ -60,7 +60,16 @@ dry-run: init
...
 mycloud: environments/environment-$(ENVIRONMENT).tfvars
-	@$(YQ) '.clouds."$(CLOUD)"' $(YQIN) < <(cat ./clouds.yaml ~/.config/openstack/clouds.yaml /etc/openstack/clouds.yaml 2>/dev/null) > mycloud.$(CLOUD).yaml
+	@if [ -f "clouds.yaml" ]; then \
+        $(YQ) '.clouds."$(CLOUD)"' $(YQIN) < clouds.yaml > mycloud.$(CLOUD).yaml; \
+    elif [ -f "~/.config/openstack/clouds.yaml" ]; then \
+        $(YQ) '.clouds."$(CLOUD)"' $(YQIN) < ~/.config/openstack/clouds.yaml > mycloud.$(CLOUD).yaml; \
+    elif [ -f "/etc/openstack/clouds.yaml" ]; then \
+        $(YQ) '.clouds."$(CLOUD)"' $(YQIN) < /etc/openstack/clouds.yaml > mycloud.$(CLOUD).yaml; \
+    else \
+        echo "Error: clouds.yaml file not found in any location."; \
```
Comment (chess-knight) [inline terraform/Makefile:70]: Lgtm, but I don't know why my make autocompletion breaks with this line:
```bash
$ make m # [tab][tab]
echo    "Error
```

Comment (fdobrovolny) [reply terraform/Makefile:70]: :thinking: I managed to reproduce it locally. I will try to look into it. The autocompletion works in general unless one tries to make autocomplete on "m"

**Review by matofeder — CHANGES_REQUESTED** [review]
The second condition does not work as expected, because of quoted tilde. Quoting of `~` (or, disables its expansion, see e.g. [here](https://stackoverflow.com/questions/67117575/how-to-use-inside-a-double-quote-path-reference)

**Review by matofeder — APPROVED** [review]
LGTM!

**Review by chess-knight — APPROVED** [review]
LGTM

**Pull request merged**